src/server/fs/lxc/webida/webida.conf

```diff
@@ -1,19 +0,0 @@
-# Template used to create this container: /usr/share/lxc/templates/lxc-ubuntu
-# Parameters passed to the template:
-# For additional config options, please look at lxc.container.conf(5)
-
-# Common configuration
-lxc.include = /usr/share/lxc/config/ubuntu.common.conf
-
-# Container specific configuration
-#lxc.rootfs = /home/webida/server-fs/lxc/webida/rootfs
-lxc.rootfs = /home/webida/webida-server/src/server/fs/lxc/webida/rootfs
-#lxc.mount = /home/sangjin/lxc/webida/fstab
-lxc.utsname = webida
-lxc.arch = amd64
-
-# Network configuration
-lxc.network.type = veth
-lxc.network.flags = up
-lxc.network.link = lxcbr0
-#lxc.network.hwaddr = 00:16:3e:aa:a2:4b
```

Comment:
It's safe. because, 
 1) lxc networking configurations are saved in /etc/lxc & /etc/default/lxc-net 
 2) rootfs path is saved in container configuration (/home/webida/lxc/webida/config) already.  It's better to respect the container config than overriding it with conf.js 
 3) <server-source/fs>/lxc, containing rootfs & container configuration, is not a part of our code but user's environment, which should be created while installing pre-requisites.